### PR TITLE
Added note about saving changed files.

### PR DESCRIPTION
**Pull request by racinmat**
Added note about saving changed files and provided link to one extension which does it for you.

### CODE REVIEW

**Comment by Majkl578** [comment]
This is potentially a security hole, when the image is in a secured part of application.

**Comment by racinmat** [comment]
yes, it is. But I think everyone who uses it is aware of how it works and that these images are accessable. But it really speeds up whole application when you do not have to boot nette for every image.

File: en/images.texy [hunk 1]
```diff
@@ -154,6 +154,8 @@ $image->send();
 $image->send(Image::PNG);
 \--
 
++.[note]
++Because rendering of image by lnk to presenter which will call $image->send(); can be little bit slow, it is better to save changed file and then to show directly this saved file. For inspiration, you can check "https://github.com/dotblue/nette-webimages":this extension.
```
Comment (dg) [inline en/images.texy]: Lnk?

Anchor syntax is "text":url

**Comment by dg** [comment]
It is dangerous.

**Comment by dg** [comment]
d001f0b08abc4ce010176eddead3926d19c96490

**Pull request closed without merging**